app/javascript/components/profile/recommendations-users.vue

```diff
@@ -17,18 +17,24 @@
         <div class="profile-recommendations-users__username">
           {{ user.login }}
         </div>
+        <div :class="`profile-recommendations-users__color-badge profile-recommendations-users__color-badge--${colorFromScore(user.score)}`"></div>
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
  This line has a length of 147. Maximum allowed is 120                   vue/max-len